app.py

```diff
@@ -1,56 +1,228 @@
+from functools import reduce
+import glob
+
 import streamlit as st
 import pandas as pd
 from PIL import Image
-import glob
+
 from src.utils import *
 
 # App UI
 st.set_page_config(
     page_title="Open Targets Data Metrics",
-    page_icon="🧊",
+    page_icon=Image.open("img/favicon.png"),
     layout="wide",
     initial_sidebar_state="expanded",
 )
 
-st.title("A sneak peek at the stats in Open Targets")
+st.title("A sneak peek at the stats in the Open Targets Platform.")
+st.markdown('##')
 st.markdown("This application is a dashboard to display the metrics of the different data releases in Open Targets.")
-st.sidebar.title("What do you want to check?")
+
+st.sidebar.header("What do you want to do?")
+page = st.sidebar.radio(
+    "Choose an option",
+    ("Explore metrics", "Compare metrics", "Visualise metrics"),
+    index=1,
+    help=("Explore metrics allows you to visualise and filter the metrics of the selected datasets. Compare metrics"
+            "allows you to compare the main metrics between two releases."))
```

Comment:
To prevent it from becoming “metricallows”:

```suggestion
    help=("Explore metrics allows you to visualise and filter the metrics of the selected datasets. Compare metrics "
          "allows you to compare the main metrics between two releases."))
```

Reply:
fixed.

Reply:
Hmm, looks like this is still not fixed in the merged version (https://github.com/opentargets/ot-release-metrics/blob/master/app.py#L27) — did you plan to add more commits?